documentation/asciidoc/topics/proc_enabling_client_cert_authn.adoc

```diff
@@ -0,0 +1,36 @@
+[id='enabling-client-certificate-authentication_{context}']
+= Enabling client certificate authentication
+
+[role="_abstract"]
+{brandname} clusters do not use client certificate authentication by default.
+Complete the following procedure so that clients must present valid certificates before they can connect to {brandname} endpoints.
+
+You can configure {brandname} to verify client identities from certificates in a trust store in two ways:
+
+`Validate`:: Requires a trust store that contains only the signing certificate, which is typically a Certificate Authority (CA). Any client that presents a signed certificate can connect to {brandname}.
```

Comment:
"Any client that presents a certificate signed by the CA can connect to {brandname}".